### PR TITLE
Wrong variables in installer's strings

### PR DESCRIPTION
**Pull request by xBorderie**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Typos! |
| Type? | improvement |
| Category? | IN |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | n/a |
| How to test? | Read the page, see if the variables work! |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: install-dev/controllers/http/configure.php [hunk 1]
```diff
@@ -97,21 +97,21 @@ public function validate()
         if ($this->session->admin_firstname && !Validate::isName($this->session->admin_firstname)) {
             $this->errors['admin_firstname'] = $this->translator->trans('Your firstname contains some invalid characters', array(), 'Install');
         } elseif (strlen($this->session->admin_firstname) > 32) {
-            $this->errors['admin_firstname'] = $this->translator->trans('The field %field% is limited to %d characters', array('field' => $this->translator->trans('firstname', array(), 'Install'), 'limit' => 32), 'Install');
+            $this->errors['admin_firstname'] = $this->translator->trans('The field %field% is limited to %limit% characters', array('field' => $this->translator->trans('firstname', array(), 'Install'), 'limit' => 32), 'Install');
```
Comment (AlexEven) [inline install-dev/controllers/http/configure.php]: %field% and %limit% in the array too ;)

**Comment by AlexEven** [comment]
Wording ok! Thank you :)

**Comment by xBorderie** [comment]
Thank you @AlexEven ! Done in https://github.com/PrestaShop/PrestaShop/pull/6148/commits/908167d4b70df5e50f1cecce56527880a1a67661 (with one additional change, plz chk).

File: install-dev/controllers/http/configure.php [hunk 2]
```diff
@@ -90,28 +90,28 @@ public function validate()
         if ($this->session->shop_name && !Validate::isGenericName($this->session->shop_name)) {
             $this->errors['shop_name'] = $this->translator->trans('Invalid shop name', array(), 'Install');
         } elseif (strlen($this->session->shop_name) > 64) {
-            $this->errors['shop_name'] = $this->translator->trans('The field %shop_name% is limited to %limit% characters', array('limit' => 64, 'shop_name' => $this->translator->trans('shop name', array(), 'Install')), 'Install');
+            $this->errors['shop_name'] = $this->translator->trans('The field %field% is limited to %limit% characters', array('limit' => 64, '%field%' => $this->translator->trans('shop name', array(), 'Install')), 'Install');
```
Comment (maximebiloe) [inline install-dev/controllers/http/configure.php]: Still need a little update here :) `limit` => `%limit%`

**Review by maximebiloe — CHANGES_REQUESTED** [review]

**Comment by maximebiloe** [comment]
@xBorderie could you update according to comment please?

**Comment by xBorderie** [comment]
@maximebiloe Done! Sorry for the delay!

**Review by maximebiloe — APPROVED** [review]

**Pull request merged**